src/Our.Umbraco.Ditto/ComponentModel/Attributes/AltUmbracoPropertyProcessorAttribute.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
```

Comment:
I know it's a preference thing, but really there is very little evidence that having them inside the namespace declaration adds any benefit, and so given the default is to be outside the namespace, I think it'd make more sense to keep them outside.